bitnami/airflow/templates/deployment-web.yaml

```diff
@@ -170,6 +174,16 @@ spec:
               {{- else }}
               value: "no"
               {{- end }}
+              {{- if .Values.airflow.pool.enabled }}
+            - name: AIRFLOW_POOL
+              value: "true"
+            - name: AIRFLOW_POOL_NAME
+              value: {{ .Values.airflow.pool.name }}
+            - name: AIRFLOW_POOL_SIZE
+              value: "32" 
+            - name: AIRFLOW_POOL_DESCRIPTION
+              value: {{ .Values.airflow.pool.description }}
+              {{- end }}
```

Comment:
The Airflow container should know about this variables. Are you doing a PR there too? I didn't see anything in your message.

Reply:
Sorry yes, I've got a pull request for the airflow image too - https://github.com/bitnami/bitnami-docker-airflow/pull/19